mubench.pipeline/tasks/implementations/publish_findings_task.py

```diff
@@ -97,9 +97,15 @@ def __slice_by_max_files_per_post(self, potential_hits: List[SpecializedFinding]
 
         yield potential_hits_slice
 
-    def _prepare_post(self, finding: SpecializedFinding, version_compile) -> Dict[str, str]:
+    def _prepare_post(self, finding: SpecializedFinding, version_compile, logger) -> Dict[str, str]:
         markdown_dict = self._to_markdown_dict(finding)
-        snippets = finding.get_snippets(version_compile.original_sources_path)
+
+        try:
+            snippets = finding.get_snippets(version_compile.original_sources_path)
+        except SnippetUnavailableException as e:
+            logger.warning("No snippets added for %s", e.file)
+            logger.debug("Failed to find snippets: %s", e.exception)
```

Comment:
Isn't `snippets` now undefined in the exception case, leading to an exception in line 109? Can we add a test for this case?